### ISSUE TITLE
PointNet code needs F.log_softmax

### ISSUE THREAD

**Issue by cory-fan** [mlvlab/PointWOLF#8]
DGCNN code ends with F.log_softmax but PointNet code does not, this results incorrectly in negative loss and poor training.
